features/filter_by_pubkey.feature

```diff
@@ -0,0 +1,149 @@
+Feature: Filter Items by Publisher Public Key
+  As any user of the HyperCore One Incubator app
+  I want to filter the list of published items by public key
+  So that I can view all items from a specific publisher
+
+
+  Background:
+    Given the following pillars are registered:
+      | name          | npub                           |
+      | PillarOne     | npub1alice789xyz...            |
+      | PillarTwo     | npub1bob456def...              |
+      | PillarThree   | npub1charlie123abc...          |
+    And the following items have been published:
```

Comment:
See https://github.com/hypercore-one/incubator/pull/5#discussion_r2265615896, https://github.com/hypercore-one/incubator/pull/4#discussion_r2270075669 on how funding applies to Projects v Tasks v Bounties.